bzr.go

```diff
@@ -146,6 +147,40 @@ func (s *BzrRepo) Tags() ([]string, error) {
 	return tags, nil
 }
 
+// CurrentVersionsWithRevs returns a list of available branches and tags, and
+// their underlying revision identifiers. The list is guaranteed to be current.
+//
+// As the model we use for bzr interaction (and go get generally does as well)
+// involves interacting only with a bzr repository's default branch, we return
+// only tags here.
+func (s *BzrRepo) CurrentVersionsWithRevs() (versions []VersionInfo, localSynced bool, err error) {
+	var out []byte
+	// Update the local first
+	_, err = s.runFromDir("bzr", "pull")
+	if err != nil {
+		return
+	}
+	localSynced = true
```

Comment:
Why is `localSynced` returned? If `bzr pull` fails it exists with no error. Wouldn't it be better to have an error representing an update failing and return that? Then you'd know a sync didn't happen. Thoughts?

This is causing me to consider the content of the error response from the `Update` method as well.